app/routes/events/view/tickets/access-codes/create.js

```diff
@@ -12,7 +12,6 @@ export default class CreateRoute extends Route {
   async model() {
     return RSVP.hash({
       accessCode: this.store.createRecord('access-code', {
-        event         : this.modelFor('events.view'),
```

Comment:
why?

Reply:
...

Reply:
https://github.com/fossasia/open-event-frontend/pull/6052 
@iamareebjamal  sir,
Just try to fix it like this. But today i will try roleback attribute on this.